src/utils/masks.ts

```diff
@@ -41,4 +41,27 @@ export const MASKS = {
   email: emailMask,
   date: [/\d/, /\d/, '/', /\d/, /\d/, '/', /\d/, /\d/, /\d/, /\d/],
   ssn: [/\d/, /\d/, /\d/, '-', /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/],
+  creditCardNumber: [
+    /\d/,
+    /\d/,
+    /\d/,
+    /\d/,
+    '-',
+    /\d/,
+    /\d/,
+    /\d/,
+    /\d/,
+    '-',
+    /\d/,
+    /\d/,
+    /\d/,
+    /\d/,
+    '-',
+    /\d/,
+    /\d/,
+    /\d/,
+    /\d/,
+  ],
+  creditCardExpiry: [/[0-1]/, /[1-9]/, '/', /[0-9]/, /[0-9]/],
```

Comment:
This mask would accept `19/99` which isn't right. If we're going to add this we should try to make it more robust.

Reply:
Expiry mask has been updated.

Reply:
@20BBrown14 please check this updated loom
https://www.loom.com/share/f55a819c4a174e738581ee1d2dcba8ad